x/group/server/invariants.go

```diff
@@ -12,13 +13,15 @@ import (
 )
 
 const (
-	votesInvariant  = "Tally-Votes"
-	weightInvariant = "Group-TotalWeight"
+	votesInvariant    = "Tally-Votes"
+	weightInvariant   = "Group-TotalWeight"
+	votesSumInvariant = "Tally-Votes-Sum"
 )
 
 func (s serverImpl) RegisterInvariants(ir sdk.InvariantRegistry) {
 	ir.RegisterRoute(group.ModuleName, votesInvariant, s.tallyVotesInvariant())
 	ir.RegisterRoute(group.ModuleName, weightInvariant, s.groupTotalWeightInvariant())
+	ir.RegisterRoute(group.ModuleName, votesSumInvariant, s.proposalTallyInvariant())
```

Comment:
could you change the method name `s.proposalTallyInvariant()` to be more consistent with the route name `votesSumInvariant`? eg `s.tallyVotesSumInvariant()`

Reply:
okay